lib/foreman_plugin_template/engine.rb

```diff
@@ -38,19 +38,17 @@ class Engine < ::Rails::Engine
       end
     end
 
-    # Precompile any JS or CSS files under app/assets/
-    # If requiring files from each other, list them explicitly here to avoid precompiling the same
-    # content twice.
-    assets_to_precompile =
-      Dir.chdir(root) do
-        Dir['app/assets/javascripts/**/*', 'app/assets/stylesheets/**/*'].map do |f|
-          f.split(File::SEPARATOR, 4).last
-        end
-      end
-    initializer 'foreman_plugin_template.assets.precompile' do |app|
-      app.config.assets.precompile += assets_to_precompile
```

Comment:
I thought this was required if you were using `rake assets:precompile` to recompile the whole application.  SETTINGS is enough if you're just using the plugin-specific asset precompilation, but for say, a source install with plugins then they'd do everything.